packages/astro/src/cli/info/index.ts

```diff
@@ -86,8 +95,9 @@ async function copyToClipboard(text: string) {
 	if (!shouldCopy) return;
 
 	try {
-		execSync(command, {
-			input: text.trim(),
+		execSync(command.replaceAll('$0', text), {
```

Comment:
A bit worried with doing a plain replacement this way that could expose executing arbitrary commands, but it looks like we don't accept any user input that reaches here, so probably fine for now.